app/javascript/controllers/text_field_controller.js

```diff
@@ -2,6 +2,7 @@ import { Controller } from 'stimulus'
 
 export default class extends Controller {
   connect () {
-    this.element.style.minHeight = `${window.innerHeight}px`
+    console.log('dispatching input', this.element)
+    this.element.dispatchEvent(new Event('input'))
```

Comment:
Why do we need this?

Reply:
Good catch. We don't use this controller.